pkg/sqlstore/impl/system/store.go

```diff
@@ -71,7 +71,7 @@ func (s *SystemStore) GetTable(ctx context.Context, id tableland.TableID) (sqlst
 		ID:      id.ToBigInt().Int64(),
 	})
 	if err != nil {
-		return sqlstore.Table{}, fmt.Errorf("failed to get the table: %s", err)
+		return sqlstore.Table{}, fmt.Errorf("failed to get the table: %w", err)
```

Comment:
switch `%s` to `%w` so we can [check](https://github.com/tablelandnetwork/go-tableland/pull/219/files#diff-495917a0bcbd30f16dabee67340414731bcf65805956d9209a21da47227885e6R73) if the query returned rows or not